hedera-node/src/main/java/com/hedera/services/store/models/Account.java

```diff
@@ -216,76 +203,53 @@ public void setNumPositiveBalances(final int numPositiveBalances) {
 	 * @param tokens
 	 * 		List of tokens to be associated to the Account
 	 * @param tokenStore
-	 * 		TypedTokenStore to validate if existing relationship with the tokens to be associated with.
+	 * 		TypedTokenStore to validate if existing relationship with the tokens to be associated with
 	 * @param isAutomaticAssociation
-	 * 		boolean flag to denote if its an automaticAssociation.
+	 * 		whether these associations count against the max auto-associations limit
 	 * @param shouldEnableRelationship
-	 * 		boolean flag to denote if the new relationships have to enabled by default without considering the KYC key and Freeze Key
+	 * 		whether the new relationships should be enabled unconditionally, no matter KYC and freeze settings
 	 * @param dynamicProperties
 	 * 		GlobalDynamicProperties to fetch the token associations limit and enforce it.
-	 * @return A list of TokenRelationships [new and old] that are touched by associating the tokens to this account.
+	 * @return the new token relationships formed by this association
 	 */
 	public List<TokenRelationship> associateWith(
 			final List<Token> tokens,
 			final TypedTokenStore tokenStore,
 			final boolean isAutomaticAssociation,
 			final boolean shouldEnableRelationship,
-			final GlobalDynamicProperties dynamicProperties) {
-		List<TokenRelationship> tokenRelationshipsToPersist = new ArrayList<>();
-
+			final GlobalDynamicProperties dynamicProperties
+	) {
```

Comment:
How are the commit interceptors used now in case of tokenAssociate and tokenDissociate transitionLogics ?